### DISCUSSION TITLE
NOUS A1T is dead after KNX config update

### DISCUSSION THREAD

**Discussion by meganom** [ascillato/Tasmota_KNX#595] [Q&A]
Hello!

I bought a pre-flashed NOUS A1T ([switchable plug with power monitoring](https://templates.blakadder.com/nous_A1T.html) ) and updated to the latest KNX version over OTA. Everything was fine. 
Later I have activated KNX and configured sending of Active Power every 10 seconds. It worked 24 H without any problems, so, i decided to add also transmission of "Energy total" and "Energy yesterday". After I added two Tx Group Addresses and clicked Save, I lost connection to the plug. After this the plug is dead, no reaction, no recovery helped. Now I am going to open it to check if I find something strange inside (maybe broken capacitors).

So, my question: have you ever faced such an issue after saving KNX settings? Or it is just a coincidence.  Thx!

Regards,
Sergey

**Comment by meganom** [comment]
Just for a record, something similar happened here: https://github.com/arendst/Tasmota/discussions/12921  and here https://github.com/arendst/Tasmota/discussions/16244

**Comment by Jason2866** [comment]
KNX is included in Tasmota main repo.
This fork is not updated and is used from the author for development of KNX in the past.
Flashing actual Tasmota with an 2 years old variant can end in a soft brick. It is a downgrade!
Downgrade is and was never supported from Tasmota!!!

**Reply by meganom** [reply]
Oh, thank you for the information.
I used this OTA link: http://ota.tasmota.com/tasmota/release/tasmota-knx.bin.gz from http://ota.tasmota.com/tasmota/release/ to update the FW. So, I guess it is a new variant, but I used maybe a wrong Repo to start a discussion? 
Info from WebGUI:
![image](https://user-images.githubusercontent.com/2887318/221041582-3482cf07-ed4f-44b0-9970-39b1d1741cd0.png)

**Comment by Jason2866** [comment]
Indeed wrong repo. https://github.com/arendst/Tasmota
